packages/valory/protocols/ledger_api/tests/test_ledger_api.py

```diff
@@ -714,6 +715,7 @@ def build_message(self) -> LedgerApiMessage:
             signed_transaction=LedgerApiMessage.SignedTransaction(
                 ledger_id=self.ledger_id, body={}
             ),
+            kwargs=Kwargs({}),
```

Comment:
effectively this will have to be done everywhere

Reply:
This is the only place where it would be required. And you've already done this

Reply:
If you're confident that it won't break anything then fine. By in principle this is a breaking change IMO.